src/app/[locale]/compare/_components/CompareView.tsx

```diff
@@ -50,6 +54,33 @@ export function CompareView() {
     const handleDeleteAas = (aasId: string) => {
         deleteAas(aasId);
     };
+
+    const handleAddAas = async (aasId: string) => {
```

Comment:
This logic was previously in ManualAasAddInput.tsx.
This should be more readable